src/benchmarktool/runscript/parser.py

```diff
@@ -314,7 +321,7 @@ def parse(self, fileName):
                 if "pbstemplate" in attr:
                     pbstemplate = attr["pbstemplate"]
                     del attr["pbstemplate"]
-                else: pbstemplate = None
+                else: pbstemplate = "templates/single.pbs"
```

Comment:
Does it really make sense to always default to this pbs template, even if you run benchmarks locally?

Reply:
I did this as a suggestion by Javier. I could change this back to None or put some other template as default. Which one would be best?

Reply:
Hi Philipp, 

This option only affect the pbsjobs, and not the seqjobs, so I do not see a problem with the case
where you run the benchmarks locally.
For me it made sense because when using the cluster I always use the same slurm template, 
and because in this way the description of the settings can be independent on whether one
will run a seqjob or a pbsjob (see the example https://github.com/kstrauch94/benchmark-tool/blob/448950da9858e7f20bbf2ee3a95be1d2aa8ad9f1/runscripts/runscript-example.xml).

Anyway, this is just a default option :)

Cheers,
Javier

Reply:
Thanks for the clarification, Javier!